src/widgets/Header.m.css

```diff
@@ -0,0 +1,195 @@
+:root {
+	--toggle-button-height: 20px;
+	--toggle-button-width: 26px;
+}
+
+.root {
+	position: fixed;
+	width: 100%;
+	background: #212329;
+	display: flex;
+	justify-content: flex-start;
+	align-content: flex-start;
+	align-items: center;
+}
+
+.root * {
+	box-sizing: border-box;
+}
+
+.menu {
+}
+
+.menu ul {
+	list-style: none;
+	display: flex;
+	padding: 0;
+	height: auto;
+	width: auto;
+	background: none;
+}
+
+.menuItem {
+	text-align: right;
+	margin: 10px;
+}
+
+.left {
+	display: flex;
+	align-items: center;
+	flex: 1 0 auto;
+}
+
+.expanded .left {
+	z-index: 999;
+}
+
+.link:hover {
+}
+
+.link {
+	font-size: 0.9rem;
+	min-width: 140px;
+	text-align: center;
+	text-decoration: none;
+	color: white;
+	box-sizing: border-box;
+	color: #999;
+	text-transform: uppercase;
+	transition: 0.1s ease-in-out;
+	transition-property: color, background-color;
+}
+
+.selected {
+}
+
+.logo {
+	height: 48px;
+	width: 110px;
+}
+
+.backdrop {}
+
+.expanded .backdrop {
+	position: fixed;
+	display: block;
+	content: "";
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 998;
+	background: rgba(0,0,0,0.85);
+	cursor: default;
+}
+
+.responsive {
+}
+
+.responsive .menu {
+	position: fixed;
+	left: -200px;
```

Comment:
Is there a way to achieve this that doesn't make use of minus positional values? (i.e. with margin etc)

Reply:
Having the negative position keeps the menu off-screen in responsive mode when it's not displayed. The `left` position is changed to `0` and animated with CSS to slide in from the left of the screen.